### ISSUE TITLE
Dialogue Logger/Parser Plugin?

### ISSUE THREAD

**Issue by gcscaglia** [exectails/MabiPale2#8]
While browsing [``MabiPale2.Plugins.PacketAnaliser.FormInfo.cs``](https://github.com/exectails/MabiPale2/blob/master/MabiPale2.Plugins.PacketAnalyzer/FrmInfo.cs) I've come across a TODO about a Dialog Parser. I couldn't find such plugin in this repository or any fork I've browsed. Is there a unpublished WIP somewhere, or was it just an idea?

I've logged a few PTJs and due to the labor involved started writing a plugin to help logging. Halfway through I've realized this plugin is in fact a limited dialogue parser of sorts, so if there is any WIP on such a plugin I could build upon it instead.

**Comment by exectails** [comment]
No, there is not dialog parser yet. I had that idea early on, but never went through with it. I probably haven't coded enough NPCs to be annoyed by copying from the analyzer yet^^

**Comment by xCryptic** [comment]
@exectails it does get kinda annoying. I actually log files with MabiPale2 and then open up the logs in the old version of MabiPale since it's right there attached (saves an extra click each time). Of course, this plugin would be more useful.

**Comment by exectails** [comment]
@xCryptic You could easily modify Pale to open the analyzer plugin each time though.
